mergify_engine/config.py

```diff
@@ -325,56 +325,65 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
 TESTING_INSTALLATION_ID: int
 SAAS_MODE: bool
 
-configuration_file = os.getenv("MERGIFYENGINE_TEST_SETTINGS")
+# config variables built
+GITHUB_DOMAIN: str
 
-if configuration_file is not None:
-    dotenv.load_dotenv(dotenv_path=configuration_file, override=True)
 
-CONFIG: typing.Dict[str, typing.Any] = {}
-for key, _ in Schema.schema.items():
-    val = os.getenv(f"MERGIFYENGINE_{key}")
-    if val is not None:
-        CONFIG[key] = val
+def load() -> typing.Dict[str, typing.Any]:
+    configuration_file = os.getenv("MERGIFYENGINE_TEST_SETTINGS")
 
+    if configuration_file is not None:
+        dotenv.load_dotenv(dotenv_path=configuration_file, override=True)
 
-# DASHBOARD API KEYS are required only for Saas
-if "SUBSCRIPTION_TOKEN" in CONFIG:
-    for key in ("DASHBOARD_TO_ENGINE_API_KEY", "ENGINE_TO_DASHBOARD_API_KEY"):
-        CONFIG[key] = secrets.token_hex(16)
+    raw_config: typing.Dict[str, typing.Any] = {}
```

Comment:
do really need `Any` here? It seems everything is a `str` in keys.

Reply:
voluptuous is untyped, so... if I put something have to ignore it everywhere after.